scripts/core/editor3/directive.tsx

```diff
@@ -342,10 +349,12 @@ class Editor3Directive {
                     }
                 })();
 
-                const renderEditor3 = () => {
+                const renderEditor3 = (options?: {unmount: boolean}) => {
                     const element = $element.get(0);
 
-                    ReactDOM.unmountComponentAtNode(element);
+                    if (options?.unmount) {
```

Comment:
You're funny :D Before, it was unmounting unconditionally. After your change it never unmounts, but you're passing arguments trying to trick me that you're preserving current behavior.